Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:2.7
+
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /code
+
+WORKDIR /code
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install -y \
+        python-dev \
+        libsasl2-dev \
+        libldap2-dev \
+        libpq-dev \
+        npm
+
+RUN npm install -g \
+        --registry http://registry.npmjs.org/ \
+        coffee-script \
+        less@1.3
+
+RUN ln -s `which nodejs` /usr/bin/node
+
+RUN pip install --upgrade pip
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY requirements-plugins.txt ./
+RUN pip install --no-cache-dir -r requirements-plugins.txt
+
+RUN pip install ipdb
```

Comment:
It's helpful for testing since the tests/building a test service use docker-compose. Without it when I test I get
```
Elaine:cabot elainearbaugh$ docker-compose run --rm web ipdb manage.py test cabot
./docker-entrypoint.sh: line 26: exec: ipdb: not found
```
(I have ipdb installed locally)